third_party/2and3/termcolor.pyi

```diff
@@ -0,0 +1,21 @@
+# Stub for termcolor: https://pypi.python.org/pypi/termcolor
+from typing import Any, AnyStr, Iterable, Optional
+
+
+def colored(
+    text: AnyStr,
+    color: Optional[str] = ...,
```

Comment:
This can actually be `unicode`, too, on Python 2. I'd recommend `Text` here, as well.